src/pages/index.hbs

```diff
@@ -1,5 +1,10 @@
 {{#extend "raha/layout/base/layout"}}
   {{#content "styles" mode="append"}}
+    h3 > a {
+      color: inherit;
```

Comment:
I don't like this. Links should be clearly links, this hides the fact that it's a link and makes it harder to understand our website. Changing that.